frameworks/helloworld/tests/test_placement.py

```diff
@@ -49,14 +49,14 @@ def fault_domain_vars_are_present(pod_instance):
 @pytest.mark.sanity
 def test_rack_not_found():
     options = {
-        'service': {
-            'spec_file': 'examples/marathon_constraint.yml'
+        "service": {
+            "spec_file": "examples/marathon_constraint.yml"
         },
-        'hello': {
-            'placement': '[["rack_id", "LIKE", "rack-foo-.*"]]'
+        "hello": {
+            "placement": "[[\\\"rack_id\\\", \\\"LIKE\\\", \\\"rack-foo-.*\\\"]]"
```

Comment:
Why are these placement changes included in this PR?

Reply:
Trying to weed my way through the Cosmos differences in 1.9 and 1.10+

Going to try this locally and see what's up.